rero_ils/modules/organisations/api.py

```diff
@@ -54,6 +54,17 @@ class Meta:
 
         default_filter = None
 
+    def get_record_by_viewcode(self, viewcode, fields=None):
+        """Search by viewcode."""
+        query = self.filter('term', code=viewcode).extra(size=1)
+        if fields:
+            query = query.source(includes=fields)
+        response = query.execute()
+        if response.hits.total.value != 1:
+            raise Exception(
+                f'Organisation viewcode {viewcode}: Result not found.')
+        return response.hits.hits[0]._source
```

Comment:
Is `_source` the same as `to_dict()`?

Reply:
No, it's a record key on elasticsearch result